testData/actions/generation/generator/UiComponentGridDataProviderGenerator/generateCollectionDataProvider/GridDataProvider.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+namespace Foo\Bar\Ui\Listing;
+
+use Magento\Framework\Api\FilterBuilder;
+use Magento\Framework\Api\Search\ReportingInterface;
+use Magento\Framework\Api\Search\SearchCriteriaBuilder;
+use Magento\Framework\Api\Search\SearchResultInterface;
+use Magento\Framework\Api\Search\SearchResultInterfaceFactory;
+use Magento\Framework\Api\SearchCriteria\CollectionProcessorInterface;
+use Magento\Framework\App\RequestInterface;
+use Foo\Bar\Model\Resource\Entity\Collection;
+use Foo\Bar\Model\Resource\Entity\CollectionFactory;
+use Magento\Framework\View\Element\UiComponent\DataProvider\DataProvider;
+
+class GridDataProvider extends DataProvider
+{
+    /**
+     * @var CollectionFactory
+     */
+    private $collectionFactory;
+
+    /**
+     * @var CollectionProcessorInterface
+     */
+    private $collectionProcessor;
+
+    /**
+     * @var SearchResultInterfaceFactory
+     */
+    private $searchResultFactory;
+
+    /**
+     * GridDataProvider constructor.
+     *
+     * @param string $name
+     * @param string $primaryFieldName
+     * @param string $requestFieldName
+     * @param ReportingInterface $reporting
+     * @param SearchCriteriaBuilder $searchCriteriaBuilder
+     * @param RequestInterface $request
+     * @param FilterBuilder $filterBuilder
+     * @param CollectionFactory $collectionFactory
+     * @param CollectionProcessorInterface $collectionProcessor
+     * @param SearchResultInterfaceFactory $searchResultFactory
+     * @param array $meta
+     * @param array $data
+     */
+    public function __construct(
+        string $name = '',
+        string $primaryFieldName = '',
+        string $requestFieldName = '',
+        ReportingInterface $reporting,
+        SearchCriteriaBuilder $searchCriteriaBuilder,
+        RequestInterface $request,
+        FilterBuilder $filterBuilder,
+        CollectionFactory $collectionFactory,
+        CollectionProcessorInterface $collectionProcessor,
+        SearchResultInterfaceFactory $searchResultFactory,
+        array $meta = [],
+        array $data = []
+    )
+    {
```

Comment:
```suggestion
    ) {
```

Reply:
Let's leave it, it caused by build-in formatted, not related to the plugin